File: BMI3.0/app/src/main/java/com/example/tipper/Question.java
package com.example.tipper;

public class Question
{
    // answerResId will store Question
    private int answerResId;

    // answerTrue will store correct answer
    // of the Question provided
    private boolean answerTrue;

    public Question(int answerResId, boolean answerTrue)
    {
        // setting the values through
        // arguments passed in constructor
        this.answerResId = answerResId;
        this.answerTrue = answerTrue;
    }

    // returning the Question passed
    public int getAnswerResId()
    {
        return answerResId;
    }

    // setting the Question passed
    public void setAnswerResId(int answerResId)
    {
        this.answerResId = answerResId;
    }

    // returning the correct answer
    // of Question
    public boolean isAnswerTrue()
    {
        return answerTrue;
    }

    // setting the correct
    // ans of Question
    public void setAnswerTrue(boolean answerTrue)
    {
        this.answerTrue = answerTrue;
    }
}

File: BMIKotlin/app/src/main/java/com/example/tipper/Question.kt
package com.example.tipper

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import com.example.tipper.R
import com.anychart.charts.Pie
import com.anychart.AnyChart
import com.anychart.chart.common.dataentry.DataEntry
import com.anychart.chart.common.dataentry.ValueDataEntry
import com.anychart.AnyChartView
import android.widget.TextView
import com.example.tipper.MainActivity
import android.widget.EditText
import android.widget.SeekBar
import android.widget.SeekBar.OnSeekBarChangeListener
import android.text.TextWatcher
import android.text.Editable
import android.content.Intent
import com.example.tipper.RecipeNr1
import com.example.tipper.RecipeNr2
import com.example.tipper.Quizz
import android.widget.ArrayAdapter
import android.widget.ImageButton
import com.example.tipper.Question
import android.annotation.SuppressLint
import androidx.annotation.RequiresApi
import android.os.Build
import android.widget.Toast

class Question     // setting the values through
// arguments passed in constructor
(// setting the Question passed
        // returning the Question passed
        // answerResId will store Question
        var answerResId: Int, // setting the correct
        // ans of Question
        // returning the correct answer
        // of Question
        // answerTrue will store correct answer
        // of the Question provided
        var isAnswerTrue: Boolean)